### ISSUE TITLE
[BUG] Ikea controller turns off Zigbee switch instead of lights

### ISSUE THREAD

**Issue by yieldhog** [xaviml/controllerx#209]
# Bug report

## Description

Hi, I have overhead lights and was able to connect my Ikea tradfri remote to Home Assistant via ZHA. In my bedroom, I have an Aqara Zigbee plug to repeat signals. I was able to identify button pushes for the remote (see log below), but, for whatever reason, anytime I push the on/off button it turns off my Zigbee plug rather than the lights. The lights I am trying to control are controlled by a Lutron wall switch integration and have no connection to the Zigbee plug. Is it possible I am putting in the controller ID from Zigbee power switch or something? Thank you

## Additional information

* Devices involved:
  * Model: IKEA E1810 as Controller
* Integration: zha
* AppDaemon version: 4
* ControllerX version: latest via HACS
* HACS version (if installed from there): latest
* Home Assistant Core version: latest

## AppDaemon app configuration

```yaml
bedroom_controller:
  module: controllerx
  class: E1810Controller
  controller: 58:8e:81:ff:fe:40:91:XX
  integration: zha
  light: light.master_bedroom_lamps
  actions:
    - "on"
    - "off"
```

## Logs

```
{
    "event_type": "zha_event",
    "data": {
        "device_ieee": "58:8e:81:ff:fe:40:91:XX",
        "unique_id": "58:8e:81:ff:fe:40:91:XX:1:0x0008",
        "device_id": "_long-id-number_",
        "endpoint_id": 1,
        "cluster_id": 8,
        "command": "step",
        "args": [
            1,
            43,
            5
        ]
    },
    "origin": "LOCAL",
    "time_fired": "2020-12-26T23:10:39.059636+00:00",
    "context": {
        "id": "_long-id-number_",
        "parent_id": null,
        "user_id": null
    }
}
```

**Comment by xaviml** [comment]
Hi @yieldhog,

I believe this is not a ControllerX problem. I would suggest you remove (or comment) the ControllerX config and see if the problem persists. I got the same problem with an E1743 IKEA controller and a Xiaomi neutral switch. There were by default in the same Zigbee group, so I had to unpair them manually with Zigbee2MQTT. You are probably experimenting with the same problem.

I currently do not have ZHA set up in my home, so I am not able to explore how to unpair devices from the same group. What I did with Z2M was to call `zigbee2mqtt/bridge/group/remove_all` topic(see [here](https://www.zigbee2mqtt.io/information/groups.html#mqtt-commands)) with the Xioami switch in the payload. I ended up creating a script to easily call it from lovelace when needed.

Regards,
Xavi M.
